specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/wsn/client.should.disconnect.if.no.control.bytes.sent/request.rpt

```diff
@@ -0,0 +1,30 @@
+connect tcp://127.0.0.1:8001
```

Comment:
Since we don't need wse tests, you can move this (and the other stuff) back up one level and remove the wsn directory.